Makefile

```diff
@@ -26,6 +26,9 @@ build:
 build_dev:
 	$(BUILD_DIR)/build.sh
 
+build_protobuf:
+	$(BUILD_DIR)/build-protobuf.sh
```

Comment:
One line command, why we need separate file?

Reply:
Support @DeV1doR, script for one line is a mental overhead.